### PR TITLE
Zero frame issue

### PR DESCRIPTION
**Pull request by davisg123**
I don't know why, but the previous logic was essentially "If an image has a zero frame, make the comparison image the same as the reference image"

### CODE REVIEW

File: Tests/SnapshotTestingTests/SnapshotTestingTests.swift [hunk 1]
```diff
@@ -825,6 +825,14 @@ final class SnapshotTestingTests: XCTestCase {
     #endif
   }
   
+  func testUIViewFailsIfNewViewHasNoFrame() {
```
Comment (PabraiVentures) [inline Tests/SnapshotTestingTests/SnapshotTestingTests.swift:828]: who tests the testing test tests?

**Review by PabraiVentures — APPROVED** [review]

File: Tests/SnapshotTestingTests/SnapshotTestingTests.swift (hunk 1 above)
Comment (davisg123) [reply Tests/SnapshotTestingTests/SnapshotTestingTests.swift:828]: giphy-2

**Comment by davisg123** [comment]
They have a ton of unreviewed PRs, but created one anyway: https://github.com/pointfreeco/swift-snapshot-testing/pull/564

File: Tests/SnapshotTestingTests/SnapshotTestingTests.swift (hunk 1 above)
Comment (oscar-tome) [reply Tests/SnapshotTestingTests/SnapshotTestingTests.swift:828]: giphy

**Pull request merged**